### ISSUE TITLE
Empty page after v0.39.0 (svelte 5)

### ISSUE THREAD

**Issue by nkitsaini** [sveltia/sveltia-cms#215]
Instead of login buttons, I see empty page. In network tab everything loads successfully and no log in console. So unable to give much helpful information. I tried latest svelte 5 version and the one from peer dependencies. Both result in empty pages. 

The page loads fine with version `0.38.4` (and previous ones), still using svelte 5 for my project.

```
// Svelte component at `/admin`
<script lang="ts">
	import { onMount } from 'svelte';
	import { get_config } from './config';
	import type { CMS } from 'decap-cms-core';

	onMount(async () => {
		// @ts-expect-error sveltia/cms doesn't have types yet. So using decap-cms-core types.
		let cms: CMS = (await import('@sveltia/cms')).default;
		// Initialize the CMS object
		cms.init({
			config: get_config(origin)
		});
	});
</script>
```

Thanks for all the amazing work!

**Comment by kyoshino** [comment]
Will check. I thought Sveltia CMS had the same issue as https://github.com/decaporg/decap-cms/issues/7223 due to Svelte’s hydration, but this issue is probably different 🤔

**Comment by kyoshino** [comment]
Fixed in [v0.39.17](https://github.com/sveltia/sveltia-cms/releases/tag/v0.39.17) 🙇🏼

**Comment by nkitsaini** [comment]
@kyoshino Nice! Made my day :D
